ob_v2p0/index.md

```diff
@@ -1,757 +1,4 @@
-
-**Steps to verify a JWS-signed badge Assertion**:
-
-1. Base64 decode the JWS payload. This will be a JSON string representation of the badge assertion.
-2. Parse the decoded JWS body into a JSON object. If the parsing operation fails, assertion MUST be treated as invalid.
-3. Perform data validation on the Assertion and the (embedded or linked) BadgeClass and issuer Profile, as well as any other relevant present data.
-4. Determine the correct signing key `id`(s) to test against the signature from the valid issuer Profile and Assertion VerificationObject. Do not trust a key that is not properly linked from the issuer Profile.
-5. Perform an HTTP GET request on the trusted keys. If it is impossible to get a usable public key, the Assertion cannot be verified.
-6. With each trusted public key, perform a JWS verification on the JWS object. If the verification fails, assertion MUST be treated as invalid.
-7. Retrieve the revocation list from the Issuer Profile object if present and ensure the `id` of the badge does not appear in the `revokedAssertions` list. Legacy v1.x badges that lack a `id` property may be identified in this list by their `uid`.
-8. If the above steps pass, assertion may be treated as valid.
-
-Other signature suites may be later included in this document if they are investigated and approved.
-
-#### Revoking a Signed Badge
-
-To mark a badge as revoked, add an entry to the resource pointed at by the Issuer Profile `revocationList` URL with the **id** of the Assertion and, optionally, a reason why the badge is being revoked. See an [example](./examples/index.html#RevocationList).
-
-
-# History <a id="History"></a>
- * [From 1.1 to 2.0](history/2.0.html) (includes 2.0 errata)
- * [From 1.0 to 1.1](history/1.1.html)
- * [From 0.5 to 1.0](https://github.com/mozilla/openbadges/wiki/Assertion-Specification-Changes)
- * [Early history of the specification](https://github.com/mozilla/openbadges-backpack/wiki/Assertions/_history)
-
-
-# List of Contributors <a id="contributors"></a>
-* Jeff Bohrer, IMS Global
-* Timothy F. Cook, LRNG
-* Maria Esquela, eNABLE Community
-* Steve Gance, Washington State Board of Community and Technical Colleges
-* Jim Goodell, QIP/CEDS
-* Markus Gylling, IMS Global
-* Viktor Haag, D2L
-* Alex Hripak, Credly
-* Kerrie Lemoie, OpenWorks Group
-* Mark Leuba, IMS Global
-* Randy Macdonald, Innovation Academy / InnovateOregon
-* Nate Otto, Concentric Sky
-* Justin Pitcher, Chalk & Wire
-* Serge Ravet, ADPIOS, BadgeEU
-* Alex Reis, D2L
-* Jarin Schmidt, Pearson Acclaim
-* Attila Szabo-Nagy, Erasmus+
+<html>
+    <head><title>Hello</title></head>
+    <body><p>Hello</p></body>
+    </html>
```

Comment:
What is the purpose of replacing the OB 2.0 document with "Hello"?

Reply:
Overall it looks like a bunch of other things are included in this PR maybe accidentally. I'm not sure what these large structural changes in the repo are about or if they're intentional. Could those be separated out into another PR so that this could be rebased to target just the changes about language?

Reply:
Modified to include only changes related to this issue